docs/components/encoder/_index.md

```diff
@@ -44,6 +44,210 @@ For configuration information, click on one of the following models:
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/encoder#Encoder).
+
+```go
+myEncoder, err := encoder.FromRobot(robot, "my_encoder")
+if err != nil {
+  logger.Fatalf("cannot get encoder: %v", err)
+}
+
+position, posType, err := myEncoder.GetPosition(context.Background(), "ticks", nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### ResetPosition
+
+Set the current position of the encoder to be the new zero position.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `extra` [(Optional[Dict[str, Any]])](https://docs.python.org/library/typing.html#typing.Optional): Extra options to pass to the underlying RPC call.
+- `timeout` [(Optional[float])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- None
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/encoder/client/index.html#viam.components.encoder.client.EncoderClient.reset_position).
+
+```python
+my_encoder = Encoder.from_robot(robot=robot, name='my_encoder')
+
+# Reset the zero position of the encoder.
+await my_encoder.reset_position()
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `cxt` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `extra` [(map[string]interface{})](https://pkg.go.dev/google.golang.org/protobuf/types/known/structpb): Extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- `error` [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/encoder#Encoder).
+
+```go
+myEncoder, err := encoder.FromRobot(robot, "my_encoder")
+if err != nil {
+  logger.Fatalf("cannot get encoder: %v", err)
+}
+
+err := myEncoder.ResetPosition(context.Background(), nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetProperties
+
+Get a list of all the position types that are supported by a given encoder.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `extra` [(Optional[Dict[str, Any]])](https://docs.python.org/library/typing.html#typing.Optional): Extra options to pass to the underlying RPC call.
+- `timeout` [(Optional[float])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- (Properties)<!-- The link in the Python SDK docs currently doesn't go anywhere useful -->: The position types supported by the encoder model.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/encoder/index.html#viam.components.encoder.Encoder.get_properties).
+
+**Example usage:**
```

Comment:
```suggestion
```
& In other places— should be clear that this is example usage without having to specify